packages/dialect-react-ui/entities/errors/ui/NotAuthorizedError.tsx

```diff
@@ -0,0 +1,44 @@
+import { useDialectWallet } from '@dialectlabs/react-sdk';
+import clsx from 'clsx';
+import { Button, Centered, ToggleSection } from '../../../components/common';
+import { H3 } from '../../../components/common/preflighted';
+import { useTheme } from '../../../components/common/providers/DialectThemeProvider';
+
+const NotAuthorizedError = () => {
+  const { textStyles } = useTheme();
+  const { hardwareWalletForced, setHardwareWalletForced, initiateConnection } =
+    useDialectWallet();
+
+  return (
+    <Centered>
+      <H3
+        className={clsx(textStyles.header, 'dt-flex dt-items-center dt-mb-4')}
+      >
+        Verify Wallet
+      </H3>
+      <span
+        className={clsx(
+          textStyles.body,
+          'dt-flex dt-items-center dt-mb-4 dt-w-[80%] dt-text-center'
```

Comment:
Why 80% width?

Reply:
AFAIK, usually containers are using 2 things: max-width and horizontal paddings/margins to adjust the container placement. 80% seems like a vague number and we won't be able to accommodate to every modal size (especially, since it's configurable). Either way, your call. Just was interested why 80%. Thanks!

Reply:
what's your suggestion?